### PR TITLE
eth/fetcher: only make noise big mismatches

### PR DESCRIPTION
**Pull request by karalabe**
Erigon is sending bad tx announces. We should drop them, but that would hurt the network. We were loud, but that hurts users. Let's lower the verbosity, get them to fix it and then raise it back up again.

### CODE REVIEW

**Review by holiman — APPROVED** [review]
LGTM

File: eth/fetcher/tx_fetcher.go [hunk 1]
```diff
@@ -593,8 +593,9 @@ func (f *TxFetcher) loop() {
 								log.Warn("Announced transaction type mismatch", "peer", peer, "tx", hash, "type", delivery.metas[i].kind, "ann", meta.kind)
 								f.dropPeer(peer)
 							} else if delivery.metas[i].size != meta.size {
```
Comment (anewt95) [inline eth/fetcher/tx_fetcher.go:595]: ****

**Review by anewt95 — COMMENTED** [review]
[]()

File: eth/fetcher/tx_fetcher.go (hunk 1 above)
Comment (Hawkins37) [inline eth/fetcher/tx_fetcher.go:595]: @anewt95

**Review by Hawkins37 — APPROVED** [review]

**Review by anewt95 — COMMENTED** [review]
{}b

**Comment by anewt95** [comment]
https://acrobat.adobe.com/id/urn:aaid:sc:VA6C2:669a4edc-a779-466f-8856-bb126b704517

**Comment by anewt95** [comment]
****

**Pull request merged**